src/lit-element.ts

```diff
@@ -53,5 +53,5 @@ export class LitElement extends UpdatingElement {
    * trigger the element to update.
    * @returns {TemplateResult} Must return a lit-html TemplateResult.
    */
-  protected render() {}
+  protected render(): TemplateResult|void {}
```

Comment:
I'd remove the `|void`. render() is supposed to be side-effect free, so `void` wouldn't make any sense.

Reply:
Yes, except that there's a non-abstract implementation of it right here, and that version returns `void`

Reply:
I wish we could implement an abstract method just so it won't throw...

Reply:
Why not make it abstract?  If that's not an option, what should it return in the cases where people extend this class but fail to implement render()?  A final option is to just `throw 'notreached'` in the impl and drop the void.